domains/namkun.json

```diff
@@ -1,9 +1,11 @@
 {
-    "owner": {
-        "username": "beocaca",
-        "email": "namdev1402@gmail.com"
-    },
-    "record": {
-        "CNAME": "namkun.pages.dev"
-    }
+  "owner": {
+    "username": "beocaca",
+    "email": "namdev1402@gmail.com"
+  },
+  "record": {
+    "CNAME": "namkun.pages.dev",
+    "MX": ["mx1.forwardemail.net", "mx2.forwardemail.net"],
+    "TXT": "forward-email=namdev1402@gmail.com"
```

Comment:
You can't have other records with `CNAME`

Reply:
[api.namkun.json](https://github.com/is-a-dev/register/pull/13849/commits/5a319f087f66ba91b31a6e19b509c8956f93a885)
can you help me preview this !!!